account-abstraction/schemas.yaml

```diff
@@ -88,30 +69,23 @@ components:
         maxSpendUsd:
           type: string
           description: Maximum amount of USD that can be spent by the Gas Manager
-          default: '6500.00'
```

Comment:
should we add nullable=true here as well?

Reply:
same comment as above!